design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,216 @@
+type Query {
+    estimateShippingCostForNegotiableQuote(
```

Comment:
`estimate` as a verb here is confusing, because it kind of makes it seem (imo) like some state-changing operation is being performed

Reply:
This has been moved under `Customer`. Thanks